WalletWasabi.Fluent/ViewModels/Wallets/Send/SendViewModel.cs

```diff
@@ -242,11 +242,9 @@ private bool TryParseUrl(string? text)
 		if (AddressStringParser.TryParse(text, _wallet.Network, out BitcoinUrlBuilder? url))
 		{
 			result = true;
-			SmartLabel label = url.Label;
-
-			if (!label.IsEmpty)
+			if (url.Label is { } label)
 			{
-				_transactionInfo.UserLabels = new SmartLabel(label.Labels);
+				_transactionInfo.UserLabels = new SmartLabel(label);
```

Comment:
Is this the same as `_transactionInfo.UserLabels = new SmartLabel(label.Labels);`?

Reply:
Yes

Reply:
Yes.
If we cast `url.Label` to `SmartLabel`, we need to access the string value by `label.Labels`.
Instead, we can simply do this above, so that `label` will have the corresponding string value that we need, without an unnecessary cast.